spring-cloud-gateway-core/src/main/java/org/springframework/cloud/gateway/filter/factory/rewrite/ModifyResponseBodyGatewayFilterFactory.java

```diff
@@ -43,24 +43,22 @@
 import static org.springframework.cloud.gateway.support.ServerWebExchangeUtils.ORIGINAL_RESPONSE_CONTENT_TYPE_ATTR;
 
 /**
- * GatewayFilter that modifies the respons body.
+ * GatewayFilter that modifies the response body.
  */
 public class ModifyResponseBodyGatewayFilterFactory extends
```

Comment:
This is a public class -  as the change is for Hoxton, we probably should not remove existing constructors as it may affect backwards compatibility.